buildbot/configure.py

```diff
@@ -125,6 +125,9 @@ def do_configure(args):
             # libclc passes `--nvvm-reflect-enable=false`, build NVPTX to enable it
             if 'NVPTX' not in llvm_targets_to_build:
                 llvm_targets_to_build += ';NVPTX'
+            # since we are building AMD libclc target we must have AMDGPU target
```

Comment:
So this is for the default build (when no specific targets have been requested), correct?  And this is for Jenkins?

As @aelovikov-intel  observed, it seems consistent with the other cases.

Reply:
This is for default build i.e. when --ci-defaults is used.
I guess we use --ci-defaults for every testing including Jenkins.